Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -556,7 +558,7 @@ void BluetoothRealDevice::SaveLinkKeys()
   std::string config_string = oss.str();
   if (!config_string.empty())
     config_string.pop_back();
-  SConfig::GetInstance().m_bt_passthrough_link_keys = config_string;
+  Config::SetBase(Config::MAIN_BLUETOOTHPASSTHROUGH_LINK_KEYS, config_string);
```

Comment:
Is `SetBase()` fine here or should this be `SetBaseOrCurrent()`? I can't really imagine a case where this is overridden by anything, but...

Reply:
SetBase sounds fine to me. We always want this to be persisted.